docs/guides/observability/keycloak-service-level-indicators.adoc

```diff
@@ -2,8 +2,8 @@
 <#import "/templates/links.adoc" as links>
 
 <@tmpl.guide
-title="{project_name} service level indicators (SLIs)"
-summary="Learn about the Service Level Indicators to monitor your {project_name} deployment's performance"
+title="Monitoring performance with Service Level Indicators"
```

Comment:
I am not sure performance is the correct word here, as SLIs are not only about performance. It kind of represents overall correctness of the system. 

Would it make sense to change it something like `Detect misbehavior [of the system] using Service Level Indicators`?

Reply:
The guide is mostly about latency and error rates that Keycloak provides, and availability that we use an external tool for (Prometheus' own `up` metric). 

As it is more in the lines of "how well the system performs" and as I would prefer a positive statement over negative statement, I'd like to stay with the term.